### PR TITLE
fixed variable naming and adjusted color of arrow highlights

### PR DESCRIPTION
**Pull request by tsevasa**

### CODE REVIEW

File: src/client/scripts/game/rendering/arrows.js [hunk 1]
```diff
@@ -482,8 +482,6 @@ const arrows = (function() {
         }
     }
 
-    function a() {return hippogonalRidersHoveredOver}
```
Comment (Naviary2) [inline src/client/scripts/game/rendering/arrows.js:501]: Whoops did I forget to delete this?

Comment (tsevasa) [reply src/client/scripts/game/rendering/arrows.js:501]: Yes!

Comment (Naviary2) [reply src/client/scripts/game/rendering/arrows.js:501]: Thank you!

Comment (Naviary2) [reply src/client/scripts/game/rendering/arrows.js:501]: (It was to help me find that bug)

**Pull request merged**